plugins/ci/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV GOOS=linux
 ENV GOARCH=amd64
 ENV trivyVersion=0.22.0
 ENV polarisVersion=1.2.1
-ENV plutoVersion=3.4.1
+ENV plutoVersion=5.0.0
```

Comment:
interesting - pluto is indeed at 5.0 as well! But I'm guessing you meant to change `polarisVersion` 😄

Reply:
very correct 😄  ... updating both